lib/index.js

```diff
@@ -8,5 +8,6 @@
  * @return {Array} An array of strings representing the text words.
  */
 module.exports = function worder(s) {
-    return s.split(/\W/g).filter(Boolean);
+    returnvar pattern = /[^a-z0-9A-Z_ÀÁÂÃÈÉÊÌÍÒÓÔÕÙÚĂĐĨŨƠàáâãèéêìíòóôõùúăđĩũơƯĂẠẢẤẦẨẪẬẮẰẲẴẶẸẺẼỀỀỂưăạảấầẩẫậắằẳẵặẹẻẽềếềểỄỆỈỊỌỎỐỒỔỖỘỚỜỞỠỢỤỦỨỪễệỉịọỏốồổỗộớờởỡợụủứừỬỮỰỲỴÝỶỸửữựỳỵỷỹ]+/;
```

Comment:
Typo: `returnvar`? Also, I guess, adding the `i` (case insensitive) flag would help.